packages/react/types/tests-css.tsx

```diff
@@ -28,6 +28,12 @@ css`
   position: relative;
   top: ${'20px'};
 `
+// $ExpectType SerializedStyles
+css({
+  ':hover': css`
+    color: hotpink;
+  `
+})
 
 // $ExpectError
 css(() => 'height: 300px;')
```

Comment:
@Andarist Are there reasons why this and below it are not allowed even though they worked?

Reply:
this `css` call doesn't run in any kind of "context", we don't have any arguments to give to this function so we don't allow functions to be passed here

Reply:
So I'm guessing, even though it works in nature, there's no point in doing this as it doesn't run in any context?
I'm looking for reasons why we do not allow these things in TS due to type complaining, but it's working just fine.